README.rst

```diff
@@ -1,205 +1,186 @@
 Papis
 =====
 
-|Build_Status| |ghbadge| |Coveralls| |RTD|
-|Code_Quality|
-|PyPI-Versions| |MYPY-CHECKED| |PyPI-Downloads|
-|Contributors| |zenodo_badge| |OpenHub|
+|Build_Status| |ghbadge| |Coveralls| |RTD| |Code_Quality| |zenodo_badge|
```

Comment:
`Build_Status` is for the Travis CI, which we're not using anymore, so that can go.

I would recommend keeping some of the PyPI badges, e.g. `|Pypi|`, `|PyPI-Versions|` and `|PyPI-Downloads|`, since installs are mostly through PyPI/pip. What do you think?

Reply:
Ok, let's do that! Which one do you prefer? I think maybe `|PyPI-Downloads|`, it conveys some information about Papis' popularity? Or alternatively `|Pypi|` as it shows that the version is up to date?

Reply:
`|Pypi|` sounds good to me, to show the version that you get from `pip install`. But if you can fit anything else on that line, then go for it!

Reply:
Hm, probably, yeah. Should wait for @alejandrogallo to confirm, but we switched to Github Actions, from what I know.

Reply:
sorry people, I was the last three  weeks on sick leave.
we are formally still using Travis I think but I think it is time to remove it, github ci is really enough. this was from the time where there was no gh ci